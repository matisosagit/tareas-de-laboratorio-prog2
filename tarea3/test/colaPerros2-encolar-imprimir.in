crearColaPerros
imprimirColaPerros
crearFecha 03/02/2024
crearPerro 1 Volta 4 10 Labrador Retriever, energético y juguetón, pelaje marrón claro
encolarColaPerros    
imprimirColaPerros
crearFecha 05/01/2024
crearPerro 9 Voltorb 5 12 Golden Retriever, amigable y leal, pelaje dorado brillante
encolarColaPerros
imprimirColaPerros
liberarColaPerros
crearColaPerros
crearFecha 20/12/2024
crearPerro 7 Luna 4 10 Husky siberiana enérgica con ojos azules que disfruta correr en la nieve
encolarColaPerros
crearFecha 21/12/2024
crearPerro 4 Turing 3 9 Un Bulldog Inglés tranquilo y cariñoso que disfruta de largas siestas y abrazos
encolarColaPerros
crearFecha 21/12/2024
crearPerro 8 Tritón 9 8 Un Poodle inteligente y juguetón con un pelaje rizado y elegante
encolarColaPerros
imprimirColaPerros
crearFecha 22/12/2024
crearPerro 3 Ampere 7 5 Un Boxer curioso y determinado, siempre explorando y metiéndose en pequeñas travesuras
encolarColaPerros
imprimirColaPerros
liberarColaPerros
Fin