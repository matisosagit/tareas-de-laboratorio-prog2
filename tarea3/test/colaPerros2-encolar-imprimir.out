1> Se ejecutó 'crearTColaPerros' exitosamente
2> Cola de Perros:
3> Fecha creada en forma exitosa.
4> 5> Se ejecutó 'encolarTColaPerros' exitosamente con perro de id 1
6> Cola de Perros:
Perro 1
Nombre: Volta
Edad: 4
Descripcion:  Labrador Retriever, energético y juguetón, pelaje marrón claro
Fecha de ingreso: 3/2/2024
Vitalidad: 10
7> Fecha creada en forma exitosa.
8> 9> Se ejecutó 'encolarTColaPerros' exitosamente con perro de id 9
10> Cola de Perros:
Perro 1
Nombre: Volta
Edad: 4
Descripcion:  Labrador Retriever, energético y juguetón, pelaje marrón claro
Fecha de ingreso: 3/2/2024
Vitalidad: 10
Perro 9
Nombre: Voltorb
Edad: 5
Descripcion:  Golden Retriever, amigable y leal, pelaje dorado brillante
Fecha de ingreso: 5/1/2024
Vitalidad: 12
11> Se ejecutó 'liberarTColaPerros' exitosamente
12> Se ejecutó 'crearTColaPerros' exitosamente
13> Fecha creada en forma exitosa.
14> 15> Se ejecutó 'encolarTColaPerros' exitosamente con perro de id 7
16> Fecha creada en forma exitosa.
17> 18> Se ejecutó 'encolarTColaPerros' exitosamente con perro de id 4
19> Fecha creada en forma exitosa.
20> 21> Se ejecutó 'encolarTColaPerros' exitosamente con perro de id 8
22> Cola de Perros:
Perro 7
Nombre: Luna
Edad: 4
Descripcion:  Husky siberiana enérgica con ojos azules que disfruta correr en la nieve
Fecha de ingreso: 20/12/2024
Vitalidad: 10
Perro 4
Nombre: Turing
Edad: 3
Descripcion:  Un Bulldog Inglés tranquilo y cariñoso que disfruta de largas siestas y abrazos
Fecha de ingreso: 21/12/2024
Vitalidad: 9
Perro 8
Nombre: Tritón
Edad: 9
Descripcion:  Un Poodle inteligente y juguetón con un pelaje rizado y elegante
Fecha de ingreso: 21/12/2024
Vitalidad: 8
23> Fecha creada en forma exitosa.
24> 25> Se ejecutó 'encolarTColaPerros' exitosamente con perro de id 3
26> Cola de Perros:
Perro 7
Nombre: Luna
Edad: 4
Descripcion:  Husky siberiana enérgica con ojos azules que disfruta correr en la nieve
Fecha de ingreso: 20/12/2024
Vitalidad: 10
Perro 4
Nombre: Turing
Edad: 3
Descripcion:  Un Bulldog Inglés tranquilo y cariñoso que disfruta de largas siestas y abrazos
Fecha de ingreso: 21/12/2024
Vitalidad: 9
Perro 8
Nombre: Tritón
Edad: 9
Descripcion:  Un Poodle inteligente y juguetón con un pelaje rizado y elegante
Fecha de ingreso: 21/12/2024
Vitalidad: 8
Perro 3
Nombre: Ampere
Edad: 7
Descripcion:  Un Boxer curioso y determinado, siempre explorando y metiéndose en pequeñas travesuras
Fecha de ingreso: 22/12/2024
Vitalidad: 5
27> Se ejecutó 'liberarTColaPerros' exitosamente
28> Fin.
